pkg/server/domain/service/user.go

```diff
@@ -94,6 +96,49 @@ func (u *userServiceImpl) Init(ctx context.Context) error {
 	return nil
 }
 
+// AdminConfigured check if admin user is initialized
+func (u *userServiceImpl) AdminConfigured(ctx context.Context) (apisv1.AdminConfiguredResponse, error) {
+	admin := model.DefaultAdminUserName
+	adminUser := &model.User{
+		Name: admin,
+	}
+	if err := u.Store.Get(ctx, adminUser); err != nil {
```

Comment:
This is less rigorous. Let's don't create the admin user by default. So, admin user initialization is required if there is no user.